packages/templates/clients/websocket/javascript/components/SendOperation.js

```
@@ -0,0 +1,30 @@
+import { Text } from '@asyncapi/generator-react-sdk';
+
+export function SendOperation({ sendOperations, clientName }) {
+  return (
+    <>
+      {sendOperations.map((operation) => (
+        <Text newLines={2} indent={2}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)